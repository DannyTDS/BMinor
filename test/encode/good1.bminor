"Hello\0x20World!"