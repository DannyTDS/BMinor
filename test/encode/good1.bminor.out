Successfully decoded and encoded string.
Input  : "Hello\0x20World!"
Decoded: Hello World!
Encoded: "Hello World!"
